content/doc/book/pipeline/jenkinsfile.adoc

```diff
@@ -944,10 +950,12 @@ Groovy string interpolation can inject rogue commands into command interpreters
 ======
 
 Another note of caution. Using Groovy string interpolation for user-controlled variables with steps that pass their arguments to command interpreters such as the `sh`, `bat`, `powershell`, or `pwsh` steps can result in problems analogous to SQL injection.
-This occurs when a user-controller variable (generally an environment variable, usually a parameter passed to the build) that contains special characters (e.g. `' " / \ $ & % ^ > < | ;`) is passed to the `sh`, `bat`, `powershell`, or `pwsh` steps using Groovy interpolation.
+This occurs when a user-controller variable (generally an environment variable, usually a parameter passed to the build) that contains special characters (e.g. `/ \ $ & % ^ > < | ;`) is passed to the `sh`, `bat`, `powershell`, or `pwsh` steps using Groovy interpolation.
 For a simple example:
 
-```
+[pipeline]
```

Comment:
this gives syntax highlighting for the jenkinsfile, and also if you wanted gives the option to add a scripted equivalent.